optimum/habana/trl/trainer/sft_trainer.py

```diff
@@ -273,6 +277,12 @@ def make_inputs_require_grad(module, input, output):
             )
             args.max_seq_length = max_seq_length
 
+        if pad_max is not None:
+            warnings.warn(
+                "You passed a `pad_max` argument to the SFTTrainer, the value you passed will override the one in the `SFTConfig`."
+            )
+            args.pad_max = pad_max
+
```

Comment:
args.pad_max will override pad_max right if  not none  ? 
pad_max will have a default false is assume and args.pad_max if set will override that.
can u pls clarify ?